### DISCUSSION TITLE
Are this glyphs of the "Private Use Area" correct?

### DISCUSSION THREAD

**Discussion by BDisp** [tui-cs/Terminal.Gui#3008] [Q&A] [answered]
![image](https://github.com/gui-cs/Terminal.Gui/assets/13117724/d8f8bfed-3467-4163-a4eb-c046f8569d12)

**Comment by tig** [comment]
What do you mean by "correct"? 

These look like nerdfont extended glyphs.

**Reply by BDisp** [reply]
Because I don't see them on https://www.compart.com/en/unicode/U+EAB0.
Writing here &#60080; we only see a square glyph. So I wonder why github doesn't recognize?

**Reply by tznind** [reply, marked as answer]
I think private use area means it is reserved range that will not be filled with official unicode symbols.  That lets custom font writers (such as nerd fonts) create and assign whatever symbols they want to those ranges.
